src/ucp/core/ucp_context.c

```diff
@@ -140,6 +140,11 @@ const char *ucp_object_versions[] = {
     [UCP_OBJECT_VERSION_LAST] = NULL
 };
 
+static const char *ucp_force_flags_names[] = {
+    [UCP_FORCE_FLAG_MULTI_SEND] = "multi_send",
+    [UCP_FORCE_FLAG_FAST_CMPL]  = "fast_cmpl",
```

Comment:
```suggestion
    [UCP_OP_ATTR_FLAG_MULTI_SEND] = "UCP_OP_ATTR_FLAG_MULTI_SEND",
    [UCP_OP_ATTR_FLAG_FAST_CMPL]  = "UCP_OP_ATTR_FLAG_FAST_CMPL",
```

1. Is that possible to use flags as indexes for names directly? If yes then code in `proto_select.c` can look like:
```c
    ucp_proto_select_param_t select_param_copy = *select_param;
    select_param_copy.op_attr |= worker->context->config.ext.force_request_flags;
```

2. I am not sure that we need to shorten flags' names here. WDYT about setting names for flags to their full version?

Reply:
1. `op_attr` in `ucp_proto_select_param` is `uint8_t`, while `ucp_op_attr_t` contains `UCP_OP_ATTR_FLAG_MULTI_SEND=UCS_BIT(19)` meaning I have to use unpack, unless I'm missing something, please check the fixes and LMKWDYT.
2. kept the short names

Reply:
we can define ucp_proto_select_op_attr_pack() as a macro, then use something like
```c
[ucs_ilog2(ucp_proto_select_op_attr_pack(UCP_OP_ATTR_FLAG_MULTI_SEND))] = "multi_send",
...
```

we can also define a macro to express `ucs_ilog2(ucp_proto_select_op_attr_pack(_value))` to shorten the code.

This way the flags will be defined with their "packed" values.
so we could just add (bitwise or) the force_op_attr_flags value into select_param->op_attr

Reply:
please check now @yosefe